src/lib/abacus/index.ts

```diff
@@ -454,8 +454,12 @@ class AbacusStateManager {
    * https://linear.app/dydx/project/feature-experimentation-6853beb333d7/overview
    */
   setStatsigConfigs = (statsigConfig: { [key in StatSigFlags]?: boolean }) => {
-    const { [StatSigFlags.ffSkipMigration]: useSkip = false } = statsigConfig;
+    const { [StatSigFlags.ffSkipMigration]: useSkip = false, ...rest } = statsigConfig;
     StatsigConfig.useSkip = useSkip;
+    Object.entries(rest).forEach(([k, v]) => {
+      // @ts-ignore
+      StatsigConfig[k] = v;
```

Comment:
Does this work? Seems dangerous to set an unknown property directly on an unknown kotlin-generated class object.